spyder/plugins/updatemanager/workers.py

```diff
@@ -152,11 +179,14 @@ def start(self):
             error_msg = HTTP_ERROR_MSG.format(status_code=page.status_code)
             logger.warning(err, exc_info=err)
         except Exception as err:
-            # Only log the error when it's a generic one because we can't give
-            # users proper feedback on how to address it. Otherwise we'd show
-            # a long traceback that most probably would be incomprehensible to
-            # them.
-            logger.warning(err, exc_info=err)
+            # Send untracked errors to our error reporter
+            error_data = dict(
+                text=traceback.format_exc(),
+                is_traceback=True,
+                title="Check for update error",
```

Comment:
```suggestion
                title="Error when checking for updates",
```

This is a bit clearer.